ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/board.c

```diff
@@ -0,0 +1,34 @@
+#include "shared-bindings/microcontroller/Pin.h"
+#include "supervisor/board.h"
+
+#include "components/driver/include/driver/gpio.h"
+
+void board_init(void) {
+    
+}
```

Comment:
```suggestion
void board_init(void) {
}
```
Remove blank spaces/empty line.